client/src/components/GoBackButton/GoBackButton.module.scss

```
@@ -0,0 +1,34 @@
+@import "../../global.scss";
+
+.goBackButton {
+    width: 2.5vw;
+    height: 4vh;
+    border-radius: 50%;
+    border: none;
+    outline: none;
+    padding: 0;
+    cursor: pointer;
+    transition: transform 0.3s, filter 0.4s;
+    filter: brightness(100%);
+    background-image: url('../../pics/goBackButton.svg');
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-color: transparent;
+
+    &:hover {
+        filter: brightness(85%);
+        transform: scale(1.1);
+        font-weight: 400;
```

Font weight auf Hover? Und dann noch ohne variable? :)
```suggestion
        font-weight: $fw-regular;
```

---

done!